x/gov/querier.go

```diff
@@ -171,14 +193,41 @@ func queryDeposits(ctx sdk.Context, path []string, req abci.RequestQuery, keeper
 	return bz, nil
 }
 
-// Params for query 'custom/gov/votes'
-type QueryVotesParams struct {
-	ProposalID uint64
+// nolint: unparam
+func queryTally(ctx sdk.Context, path []string, req abci.RequestQuery, keeper Keeper) (res []byte, err sdk.Error) {
```

Comment:
imho, we should remove the named return vars here (it's obvious what it's returning) and then we can get rid of all the `err2`s.

Reply:
fixed